src/main/java/com/yugabyte/sample/apps/SqlDataLoad.java

```diff
@@ -161,7 +163,11 @@ public String getTableName() {
 
     private PreparedStatement getPreparedSelect() throws Exception {
         if (preparedSelect == null) {
-            preparedSelect = getPostgresConnection().prepareStatement(
+            if (selConnection != null) {
```

Comment:
close() will check whether selConnection is null, so can remove this check, same for line 227, and in corresponding functions in SQLInsert.java

Reply:
Comment addressed.